index.md

```diff
@@ -43,9 +43,9 @@ title: DreamHost Design System
 	<li>1) Design a product or page using the existing Sketch pattern library.</li>
 	<li>2) Build product prototype &amp; production code using DreamHost.css existing components creating custom styles where needed for new components.</li>
 	<li>3) Release &amp; prove new components.</li>
-	<li>3) Add new components to Sketch pattern library.</li>
-	<li>4) Build new components into DreamHost.css framework.</li>
-	<li>5) Document new components and variants.</li>
+	<li>4) Add new components to Sketch pattern library.</li>
```

Comment:
I feel like this should be an `<ol>` instead of an `<ul>` that way inconsistencies like what caused this fix would be a non-issue.

Reply:
Good point but <ol> has styles removed in DreamHost.css so numbers don't appear.

Reply:
Can we add it? doesn't seem like that would be a huge job, no? Even if we just adopt the current <ul> styles for now, but so that ordered lists also work. I can see that being a necessary thing down the line anyway. May as well knock it out.

Reply:
okay sounds good, will update to `<ol>` ahead of that change to the framework repo